### PR TITLE
remove duplicate key from oas

### PR DESCRIPTION
**Pull request by pKorsholm**
**What**
- remove duplicate oas key

**Why**
- oas generation fails when duplicate keys are present

### CODE REVIEW

File: packages/medusa/src/models/product-category.ts [hunk 1]
```diff
@@ -94,9 +94,8 @@ export class ProductCategory extends SoftDeletableEntity {
  *       type: object
  *       description: A product category object.
  *   parent_category:
- *     description: Available if the relation `parent_category` is expanded.
+ *     description: A product category object. Available if the relation `parent_category` is expanded.
```
Comment (adrien2p) [inline packages/medusa/src/models/product-category.ts:97]: Question: is it not returned by default? We have to pass it to the expand query params to get it?

**Review by adrien2p — APPROVED** [review]
LGTM 😊

File: packages/medusa/src/models/product-category.ts (hunk 1 above)
Comment (riqwan) [reply packages/medusa/src/models/product-category.ts:97]: Thanks for fixing this! Its not eager loaded by default, but this is for all potential options for the model right?

Comment (adrien2p) [reply packages/medusa/src/models/product-category.ts:97]: I am asking myself if the expand notion should even appear in the model description. It is an api query param. Wdyt?

**Pull request merged**